<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,1">
        <item>
         <widget class="QPushButton" name="btnTestConnection">
          <property name="text">
           <string>接続テスト</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>デバイス</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbDevice"/>
        </item>
        <item>
         <widget class="QLabel" name="txtDevice">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QSplitter" name="splMainV">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splMainH">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,1,0,0,0">
         <item>
          <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0,0,0" columnstretch="0,0,1,0">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>書き込み</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="btnWrite">
               <property name="text">
                <string>書き込み</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="3">
            <widget class="QPushButton" name="btnSelectFlashWrite">
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="btnFlashRead">
               <property name="text">
                <string>FLASH読み出し</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="2">
            <widget class="QLineEdit" name="edtFlashRead"/>
           </item>
           <item row="5" column="3">
            <widget class="QPushButton" name="btnSelectFlashRead">
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="edtFlashWrite"/>
             </item>
            </layout>
           </item>
           <item row="2" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="edtEEPWrite"/>
             </item>
            </layout>
           </item>
           <item row="2" column="3">
            <widget class="QPushButton" name="btnSelectEEPWrite">
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>読み出し</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>13</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>FLASH</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QPushButton" name="btnEEPRead">
             <property name="text">
              <string>EEPROM読み出し</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>EEPROM</string>
             </property>
            </widget>
           </item>
           <item row="6" column="2">
            <widget class="QLineEdit" name="edtEEPRead"/>
           </item>
           <item row="6" column="3">
            <widget class="QPushButton" name="btnSelectEEPRead">
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="chkEEPWrite">
             <property name="text">
              <string>EEPROM</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>FLASH</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="chkVerify">
             <property name="text">
              <string>照合</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>13</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>ターゲット通信</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,0,1,0">
           <item>
            <widget class="QPushButton" name="btnExecTarget">
             <property name="text">
              <string>ターゲット起動</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnResetTarget">
             <property name="text">
              <string>リセット</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QGridLayout" name="grdPins">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="loSendMessage">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>送信</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtSendMessage"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,0,0">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>ヒューズ</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnWriteFuse">
             <property name="text">
              <string>書き込み</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnReadFuse">
             <property name="text">
              <string>読み出し</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="scrFuses">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>253</width>
              <height>252</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="scrFusesLayout"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="loConsole">
          <item>
           <widget class="QTextEdit" name="edtConsole"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>表示項目</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkShowReceived">
            <property name="text">
             <string>受信メッセージ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkShowSended">
            <property name="text">
             <string>送信エコー</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkShowAppMessage">
            <property name="text">
             <string>アプリケーションメッセージ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chkShowError">
            <property name="text">
             <string>エラー</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearConsole">
            <property name="text">
             <string>クリア</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>737</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>ヘルプ(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="actionConfig"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_S"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConfig">
   <property name="text">
    <string>設定</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>終了</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>このソフトについて</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
